config/config.go

```diff
@@ -124,7 +124,8 @@ type Config struct {
 	DisableMetrics       bool
 	PrometheusRegisterer prometheus.Registerer
 
-	NoDelayNetworkDialRanker bool
+	SmartDialing       bool
```

Comment:
Why is this a bool? How would a user set a different dial ranker than the default one.

Reply:
I made it a yes no options. Have changed this to allow custom dial ranker implementations.